go.mod

```
@@ -3,8 +3,8 @@ module github.com/edgexfoundry/device-virtual-go
 go 1.20
 
 require (
-	github.com/edgexfoundry/device-sdk-go/v3 v3.0.0-dev.54
-	github.com/edgexfoundry/go-mod-core-contracts/v3 v3.0.0-dev.28
+	github.com/edgexfoundry/device-sdk-go/v3 v3.0.0-dev.57
```

This will allow the removal of TOML module from the Attribution.txt file

```suggestion
	github.com/edgexfoundry/device-sdk-go/v3 v3.0.0-dev.60
```

---

@cloudxxx8 I will wait for your fix to continue this toml to yaml conversion.

---

This pr has to be merged first

---

ok, I will leave it as using dev57, it is ready to be merged